code/game/jobs/job/civilian.dm

```diff
@@ -383,7 +383,7 @@
 	idtype = /obj/item/weapon/card/id/centcom
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "NanoTransen Law, CentComm Officals, and the stations captain."
+	supervisors = "Nanotrasen Law, CentComm Officals, and the stations captain."
```

Comment:
Such is the curse of any fix. You end up fixing 4x as many **oldcoder** mistakes as you do your own. Just as @PJB3005. Every time I refactor something I end up having to delete trailing returns 400 lines up in the file away from where I made the change.